src/resources/PanelLayout.py

```diff
@@ -0,0 +1,24 @@
+from rich.layout import Layout
+
+
+class PanelLayout:
+    """Define panel layout"""
+
+    def __init__(self) -> None:
+        pass
+
+    def make_layout() -> Layout:
+        """Makes the layout"""
+        layout = Layout(name="root")
+
+        layout.split_row(
+            Layout(name="main", ratio=6),
+            Layout(name="tree", ratio=2),
+        )
+
+        layout['main'].split_column(
+            Layout(name='main game', ratio=7),
```

Comment:
With the suggested changes earlier:
```suggestion
            Layout(name='main_game', ratio=7),
```

That's the end of my review, great work with this!